src/components/approval-tasks/ApprovalRow.scss

```diff
@@ -0,0 +1,8 @@
+.odc-pl-approval-status-icon {
```

Comment:
I don't know if we have more `odc-pl` prefixes, but I think we should prefix them with something like `pipelines-plugin-` instead of odc for the dynamic plugin. Wdyt @Lucifergene @vikram-raj @lokanandaprabhu ?

Reply:
prefixed with `pipelines-` for now :)